[8.6] Adds osquery guides known issue to 8.6 release notes (#2887)
elastic/security-docs: docs/release-notes/8.6.asciidoc

[discrete]
[[known-issue-8.6.0]]
==== Known issues

> * Osquery investigation guides may not render formatting correctly if they include an escape: `\"`. You should copy-paste affected queries directly rather than importing them to avoid breaking query syntax (https://github.com/elastic/detection-rules/pull/2447[#2447]).

Reviewer: +1 @nastasha-solomon and a few more edits:
```suggestion
* Osquery investigation guides may not render correctly if they include an escaped character (such as `\"`). To avoid breaking the query syntax, copy and paste affected queries directly into the investigation guide instead of importing them (https://github.com/elastic/detection-rules/pull/2447[#2447]).
```

> * Osquery investigation guides may not render formatting correctly if they include an escape: `\"`. You should copy-paste affected queries directly rather than importing them to avoid breaking query syntax (https://github.com/elastic/detection-rules/pull/2447[#2447]).

Reviewer 2: @joepeeples @nastasha-solomon 
Please change the wording, it is not reflecting the issue and fix correctly.

Suggestion:
Some pre-built rules investigation guides may not render correctly if they include an escaped character (such as `\"`). The issue will be resolved when you update the pre-built rules when prompted to from the UI. (https://github.com/elastic/detection-rules/pull/2447[#2447]).
